packages/bot-skeleton/src/services/api/active-symbols.js

```diff
@@ -178,7 +178,7 @@ export default class ActiveSymbols {
             submarket_options.sort(a => (a[1] === 'random_index' ? -1 : 1));
         }
 
-        return this.sortDropdownOptions(submarket_options, this.isSubmarketClosed);
+        return this.sortDropdownOptions(submarket_options.sort(), this.isSubmarketClosed);
```

Comment:
We have sortDropdownOptions to check close market and open market, If you sort it here still we have another order based on close markets and open markets

Reply:
added sort to sortdropdown options function